docs/rfcs/014-born_digital_workflow/README.md

```diff
@@ -0,0 +1,176 @@
+
+This metadata is used for identification only. The canonical version of the metadata, which is also the data that will be displayed to end users, remains the data in Calm. Similarly, whilst Archivematica does have support for rights information, we will not be supplying this through the workflow. Calm will be the sole location for rights information.
+
+The package created for transfer into Archivematica should reflect the original order of the files, as this structure will be maintained in the storage service and in the file-level metadata presented to users.
+
+The zip file should have the following structure:
+
+	PPABC_1234.zip
+	|-- file1.txt
+	|-- file2.doc
+	|-- file3.jpg
+	|-- folder1
+	    \-- [...] 
+	|-- folder2
+	    \-- [...] 
+	|-- [...]
+	\-- metadata
+	    \-- metadata.csv
+
+ Note the addition of a `metadata` directory, with a single file named `metadata.csv`, which contains item level metadata from Calm. `metadata.csv` should have the following structure:
+
+	filename,dc.title,dc.creator,dc.date,dc.identifier
+	objects,<Title>,<CreatorName>,<Date>,<RefNo>
+
+Once the zip files have been created, they need to be uploaded to AWS S3. As soon as the upload of a zip completes, it should be picked up by Archivematica and a processing workflow started automatically.
+
+Access to S3 requires the installation of a client application. Until this can be made available more widely, zip files should be passed to the Digital Production team for upload to S3.
+
+| Step                 | Description |
+|----------------------|-------------|
+| Create packages      | Move the set of files for each item into its own directory, preserving the original order. |
+| Add package metadata | Add a `metadata.csv` file to each item, inside a directory called `metadata`. This file should include the appropriate item level metadata from Calm. |
+| Upload packages      | Create a separate zip file from the parent directory for each item and upload to AWS S3, via the Digital Production team. |
+
+### Processing
+
+Archivematica will be configured to perform all processing steps automatically. No manual intervention should be required once zip files created with the current manual appraisal process have been uploaded to S3.
+
+We will use the default format policies and tools that come pre-configured in Archivematica, ensuring ongoing best practice for format identification, metadata extraction and file normalisation. All files are virus checked by Archivematica before processing.
+
+| Step                  | Description |
+|-----------------------|-------------|
+| Format identification | Determine the most reliable way to identify the format of each file, as described in PRONOM. |
+| Metadata extraction   | Run format-specific tools to extract as much metadata as possible from within each file. |
+| File normalisation    | Convert files into the formats best suited for preservation and access, in addition to keeping the original file. |
```

Comment:
What sort of reporting is possible on this? As part of the digital audit we know that we need to keep better records about our digital material. It would be very useful to be able to add a record of this normalisation process to the file for audit purposes.